database/postgres/postgres_test.go

```diff
@@ -508,5 +508,40 @@ func Test_computeLineFromPos(t *testing.T) {
 			run(true, true)
 		})
 	}
+}
+
+func Test_quoteIdentifier(t *testing.T) {
```

Comment:
Use [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests)

Reply:
@dhui I have pushed a new version with `table driven tests`.

Is it ok for you?